pennylane/templates/broadcast.py

```diff
@@ -549,7 +549,10 @@ def circuit(pars):
 
         # expand dimension so that parameter sets for each unitary can be unpacked
         if len(shape) == 1:
-            parameters = qml.tape.interfaces.functions.WrapperFunctions.expand_dims(parameters, 1)
+            if qml.tape_mode_active():
+                parameters = qml.tape.UnifiedTensor(parameters).expand_dims(1).data
+            else:
+                parameters = [[p] for p in parameters]
```

Comment:
So basically from now on we would just wrap everything we need to manipulate with a unified tensor? Sounds good!

Reply:
Yep! And remember to call `.data` at the end, if we want the raw unwrapped tensor